### ISSUE TITLE
Provide human-readable version(s) of this "spec"

### ISSUE THREAD

**Issue by olyerickson** [okfn/data-catalog-spec#2]
An executable "spec" is very clever, but is not what the user expects when following a link such as "Check out our spec!"

Please provide at least PDF version of the most recent versions, and/or a README to the uninitiated as to what they are landing on/in when they arrive here. 

Especially if you plan to publicize this far and wide, as you are...

Thanks!

**Comment by band** [comment]
I agree with John Erickson re providing human readable versions of the spec or some documentation about how such a document is produced.

That said, providing source code for the spec is a good idea.

Cheers.

**Comment by rufuspollock** [comment]
@olyerickson just to clarify is that regarding this source repo or http://spec.datacatalogs.org/ - the latter is what most people should be looking at. You only look at the "source" text in this repo if you want to modify.
